app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -109,6 +109,9 @@ class ApplicationConfig @Inject()(
 
   lazy val citizenAuthFrontendSignOutUrl: String = citizenAuthHost + "/ida/signout"
 
+  lazy val sessionTimeoutInSeconds: Int = 900
+  lazy val sessionCountdownInSeconds: Int = 120
```

Comment:
Can we read these from a config? We sometimes want to change it on different environments to help testing

Reply:
done